v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+| report_id               | [`INT8`](int.html) | The ID of the [report](#system-reports_meta) that generated this data.                 |
+| total_ranges            | [`INT8`](int.html) | Total [ranges](architecture/overview.html#architecture-range) in the cluster.          |
+| unavailable_ranges      | [`INT8`](int.html) | Unavailable ranges.                                                                    |
+| under_replicated_ranges | [`INT8`](int.html) | [Under-replicated](admin-ui-replication-dashboard.html#under-replicated-ranges) ranges. |
+| over_replicated_ranges  | [`INT8`](int.html) | Over-replicated ranges.                                                                |
+
+### system.replication_critical_localities
+
+The `system.replication_critical_localities` table shows which localities in your cluster have data that is at risk in the event of node failures in your cluster.
+
+For an example using this table, see [Find out which databases and tables have at-risk ranges](#find-out-which-databases-and-tables-have-at-risk-ranges).
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW COLUMNS FROM system.replication_critical_localities;
+~~~
+
+| Column name    | Data type               | Description                                                                                                             |
+|----------------+-------------------------+-------------------------------------------------------------------------------------------------------------------------|
+| zone_id        | [`INT8`](int.html)      | The ID of the [replication zone](configure-zone.html).                                                                  |
+| subzone_id     | [`INT8`](int.html)      | The ID of the subzone (i.e., [partition](partition-by.html)).                                                           |
+| locality       | [`STRING`](string.html) | The human-readable name of the critical [locality](configure-replication-zones.html#zone-config-node-locality).         |
+| report_id      | [`INT8`](int.html)      | The ID of the [report](#system-reports_meta) that generated this data.                                                  |
+| at_risk_ranges | [`INT8`](int.html)      | The [ranges](architecture/overview.html#architecture-range) that are at risk of data loss as of the time of the report. |
+
+{{site.data.alerts.callout_info}}
+If you have not [defined any localities](configure-replication-zones.html#zone-config-node-locality), this report will not return any results. It only reports on localities that have been explicitly defined.
+{{site.data.alerts.end}}
+
+### system.replication_constraint_stats
+
+The `system.replication_constraint_stats` table lists violations to any data placement requirements you've configured, along with information about whether the cluster is actively recovering from those violations.
```

Comment:
again I'd scratch the thing about recovering

Reply:
Removed all references to recovering here and in other places